### DISCUSSION TITLE
403 Forbidden from POST

### DISCUSSION THREAD

**Discussion by poetsmeniet** [OP-TED/eForms-SDK#511] [Q&A] [answered]
I have an enotices2 account, and logged in to the Ted Developer portal. I created API keys on both environments. On preview and prod environments I get the following error using the Swagger UI:

Request: search-esentool
403 Forbidden from POST http://uaa.teden2.reception.internal.op.aws.cloud.tech.ec.europa.eu/auth/internal

I read the API documentation, but can't find anything that I'm missing. What could I be doing wrong?

**Comment by rouschr** [comment, marked as answer]
Hello,

for questions related to enotices2 / teden2, please use the helpdesk of enotices2:
https://enotices2.ted.europa.eu/help/

**Comment by klakegg** [comment]
For anyone else coming by this issue, please see issue [#323](https://github.com/OP-TED/eForms-SDK/discussions/323).
